libs/base/Data/Vect.idr

```diff
@@ -267,16 +267,13 @@ zipWith f (x::xs) (y::ys) = f x y :: zipWith f xs ys
 
 ||| Linear version
 public export
-lzipWith : (f : (1 _ : a) -> (1 _ : b) -> c)
-      -> (1 _ : Vect n a)
-      -> (1 _ : Vect n b)
-      -> Vect n c
+lzipWith : (f : a -> b -> c) -> Vect n a -> Vect n b -> Vect n c
```

Comment:
This is just `zipWith` now. So we should either give it back its type
or remove it entirely IMO.

Reply:
Ah yes, good spot. `relax` seems like it could be useful too, and if it is, it probably belongs in the prelude.